.travis.yml

```diff
@@ -35,12 +33,6 @@ script:
   - py.test --cov-append --cov-report=term --cov=. -v --create-db --migrations
   - npm test
 
-  # Run all frontend checks
-  - pushd frontend
-  - npm run flow
-  - npm test
-  - popd
-
   - codecov
```

Comment:
I think you can remove this one too?

Reply:
That'll come when I remove the pytest

Reply:
I though you were already running it at the end of `frontend.yml`. Are those two different things? I noticed a coverage report at the end of pytest, but assumed it was one of the arguments passed.